.gitignore

```diff
@@ -15,3 +15,5 @@ dist/
 
 # Environments
 .env
+Pipfile
```

Comment:
As Pipfile is excluded and removed from the project, no need to keep in `.gitignore`.